src/main/java/lotto/util/StringUtil.java

```diff
@@ -0,0 +1,15 @@
+package lotto.util;
+
+public class StringUtil {
+    public static String[] split(String delimiter, String inputString) {
+        if (isIllegalArgument(inputString)) {
+            throw new IllegalArgumentException();
+        }
+        return inputString.replace(" ", "")
+                .split(delimiter);
+    }
+
+    private static boolean isIllegalArgument(String inputString) {
+        return inputString == null || "".equals(inputString.trim()) ? true : false;
```

Comment:
"".equals(inputString.trim()) 검증으로도 원하시는 리턴값이 나올 것 같습니다! 

```suggestion
        return inputString == null || "".equals(inputString.trim());
```